core/src/jarFileTest/java/org/testcontainers/PublicBinaryAPITest.java

```diff
@@ -41,9 +41,9 @@ public class PublicBinaryAPITest extends AbstractJarFileTest {
     private static String SHADED_PACKAGE_PATH = SHADED_PACKAGE.replaceAll("\\.", "/");
 
     static {
-        Assertions.registerFormatterForType(ClassNode.class, it -> it.name);
-        Assertions.registerFormatterForType(FieldNode.class, it -> it.name);
-        Assertions.registerFormatterForType(MethodNode.class, it -> it.name + it.desc);
+        registerFormatterForType(ClassNode.class, it -> it.name);
+        registerFormatterForType(FieldNode.class, it -> it.name);
+        registerFormatterForType(MethodNode.class, it -> it.name + it.desc);
```

Comment:
not sure about this change, let's maybe revert? Makes it hard to follow who defines `registerFormatterForType`